src/Infrastructure/Array/interface/ESMF_ArrayGet.cppF90

```diff
@@ -52,6 +52,7 @@ module ESMF_ArrayGetMod
   use ESMF_DistGridMod
   use ESMF_RHandleMod
   use ESMF_F90InterfaceMod  ! ESMF Fortran-C++ interface helper
+  use ESMF_InfoMod
```

Comment:
Unclear why this line is needed here.

Reply:
Nice catch. I forgot to remove it when there was some thought of adding info gets to all the object interfaces. I pushed a fix.